### PR TITLE
Add `LocalStorageListenerService` to logout from other tabs

### LINKED ISSUE #19488: Feature: Log out from other tabs automatically when user logout from the current tab
- [x] Angular
- [x] Blazor
- [x] MVC

### PR DESCRIPTION
**Pull request by Sinan997**
Resolves #19488

### CODE REVIEW

**Comment by masum-ulu** [comment]
Can we move this service to core package 🤔 ?

**Comment by Sinan997** [comment]
> Can we move this service to core package 🤔 ?

I am not sure about it, do we really want to depend on `"angular-oauth2-oidc"` package in `core` package?

**Comment by masum-ulu** [comment]
> > Can we move this service to core package 🤔 ?
> 
> I am not sure about it, do we really want to depend on `"angular-oauth2-oidc"` package in `core` package?

:') we don't need to depend it, we can use IAuthService abstraction class right ? App project will create instance with oauth module configuration

File: npm/ng-packs/packages/core/src/lib/services/local-storage-listener.service.ts [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
 })
 export class LocalStorageListenerService {
   protected readonly window = inject(DOCUMENT).defaultView;
-  protected readonly authService = inject(OAuthService);
 
   constructor() {
     this.window.addEventListener('storage', event => {
       if (event.key === 'access_token' && event.newValue === null) {
-        this.authService.logOut();
+        this.window.location.reload();
```
Comment (masum-ulu) [inline npm/ng-packs/packages/core/src/lib/services/local-storage-listener.service.ts:13]: We've AuthService in core package try to use it please

Comment (Sinan997) [reply npm/ng-packs/packages/core/src/lib/services/local-storage-listener.service.ts:13]: I have tried it doesn't work as expected.

Reloading other tabs is giving us what we want to achieve. So no need to use `AuthService`

**Comment by masumulu28** [comment]
Maybe we can use [AbpLocalStorageService](https://github.com/abpframework/abp/blob/dev/npm/ng-packs/packages/core/src/lib/services/local-storage.service.ts)

**Review by masum-ulu — APPROVED** [review]

**Review by oykuermann — APPROVED** [review]

**Pull request merged**